apps/src/lib/kits/maker/ui/SetupChecklist.jsx

```diff
@@ -273,89 +286,44 @@ export default class SetupChecklist extends Component {
             {this.state.caughtError && this.state.caughtError.reason && (
               <pre>{this.state.caughtError.reason}</pre>
             )}
-            We couldn't detect a Circuit Playground board. Make sure your board
-            is plugged in, and click{' '}
+            {applabI18n.makerSetupPlugInBoardCheck()}
             <a href="#" onClick={this.redetect.bind(this)}>
-              re-detect
+              {applabI18n.redetect()}
```

Comment:
[incorporate these strings into 1 for i18n]

Reply:
Hi there! Can you let me know which string parts you are looking at here? Thanks!

Reply:
of course -- i was thinking we could condense `applabI18n.makerSetupPlugInBoardCheck` and `applabI18n.redetect` using the `<SafeMarkdown/>` component, but now i realize that won't work because the `onClick` handler here won't play nicely with markdown, so you can ignore this one 😄